woocommerce/woocommerce-android
# UOE Refactor names

Part of: #6613

### Description
This PR refactor all class names from OrderCreation..-> OrderCreateEdit... so that they are not only valid for order creation.

### Testing instructions

- Run all unit tests in WooCommerce and check that all pass 
- Smoke test order creation and order editing and check nothing breaks
